Library/Homebrew/extend/pathname.rb

```diff
@@ -29,7 +29,7 @@ def compute_disk_usage
     if self.directory?
       @file_count = 0
       @disk_usage = 0
-      self.find do |f|
+      find do |f|
```

Comment:
And removed the `self` from `self.directory?`. Doesn't seem to break anything. `brew cleanup` still functions as expected, and `brew tests` pass fine.